rskj-core/src/main/java/co/rsk/peg/FederationUtils.java

```diff
@@ -0,0 +1,22 @@
+package co.rsk.peg;
+
+import co.rsk.bitcoinj.script.Script;
+import co.rsk.peg.bitcoin.Standardness;
+
+import static co.rsk.peg.FederationCreationException.Reason.ABOVE_MAX_SCRIPT_ELEMENT_SIZE;
+
+public class FederationUtils {
+    private FederationUtils() {
+    }
+
+    public static void isScriptSizeValid(Script script) throws FederationCreationException {
```

Comment:
Nit picky, but since it's no longer a boolean method I would not call `is...`. That gives me the impression that it will return true or false. I'd simply call it `validateScriptSize` or something like that